### ISSUE TITLE
Conform AgeRating to ComicInfo schema

### ISSUE THREAD

**Issue by Majawat** [mylar3/mylar3#1213]
**Describe the request**
ComicInfo's schema has different values for the AgeRating value as seen here: https://github.com/anansi-project/comicinfo/blob/main/drafts/v2.1/ComicInfo.xsd instead of the values that Mylar currently support.

**How did you come up with the idea?**
Komga conforms to the ComicInfo schema as described above, thus trying to use Mylar generated values fail to be imported into Komga. https://github.com/gotson/komga/issues/885

**Environment (please complete the following information):**
 - OS: Docker on unraid via Community Plugins
 - Branch / commit: Version: v0.6.7 (master)

**Additional information**
(pretty please?)

**Comment by barbequesauce** [comment]
AgeRating is not calculated by Mylar, it's a data point from ComicVine. I'd suggest submitting wiki edits to CV for the issues you're having concerns with....

**Comment by Majawat** [comment]
Sorry, I'm not seeing where ComicVine shows an Age (or Content) Rating anywhere (https://comicvine.gamespot.com/the-uncanny-x-men/4050-3092/ for example), nor do I see how Mylar grabs it from CV, as I don't see any Fields in their API: https://comicvine.gamespot.com/api/documentation.

As a result, I'm also noticing that none of the Comics have the Age Rating populated with anything beyond "None". Am I missing something?

**Comment by evilhero** [comment]
Age Rating is a manual field at the series level - meaning you have to change it from None to one of the listed values.

Once set it applies to all subsequent issues tagged within mylar for that series. 

Age Rating even in CT is a manual field, as CV doesn't auto-populate it as that information is not available (outside of using CR plugins).

The age rating values were not present in the original comicinfo schema previously to any set standard, so I had adopted the comixology ratings almost 2 years ago (which look how well that turned out lol).

I'll look into it - my only concern being it will not be backwards compatibile with users files / series.json who have tagged their issues/series previously. My priority is always mylar usage vs 3rd party usage as far as compatibility goes.

**Comment by Majawat** [comment]
> Age Rating is a manual field at the series level - meaning you have to change it from None to one of the listed values.

Ok, gotcha. That's what I thought. 

> The age rating values were not present in the original comicinfo schema previously to any set standard, so I had adopted the comixology ratings almost 2 years ago (which look how well that turned out lol).

I think you did the best you could have with the info given at the time. I especially really like the values you chose, they're clear and concise!

> I'll look into it - my only concern being it will not be backwards compatibile with users files / series.json who have tagged their issues/series previously. My priority is always mylar usage vs 3rd party usage as far as compatibility goes.

I absolutely understand about being backwards compatible/non-breaking. After sleeping on it myself, I think there's two courses of actions that fit everyone's needs. 

1. Mylar adds the current schema values without removing the existing values. This would make Mylar's ComicInfo non-conforming, but would give users most flexibility.
2. Petition ComicInfo to add Mylar's current values to the schema. I personally prefer this option, as the current schema values feel clunky and disorganized. 

Both of the above could be done too: add Mylar's values to the schema, then Mylar adds all of the 'new' values. Making everything sync. 

The third option of just conforming to current standard would break exciting data. But have options there:

1. Automatically convert existing values to the schema equivalent. You'd probably need to figure out a way to inform users before hand, else lots of pitchforks.
2. Don't automatically convert, but maybe have a one-time setting after the update to map the current values to the schema equivalent, then have a button that goes through the library and updates values, and writes the metadata. Gives options, but still many pitchforks.

Neither of those are elegant. And I'm sure there's other options. But just what I've came up with so far. 

All in all, I love Mylar, it truly is the best software for what it does. I can't express how much I appreciate your hard work and being open to new ideas!

**Comment by evilhero** [comment]
Yeah the main problem is that you have actors that are tyring to manipulate things to their way of thinking, and not being open to any change that conflicts with their supposed view of the schema.

The project you linked is not a standard by any stretch, altho what it seems like they're doing is trying to pigeonhole people into using what they feel it should be and ignoring opinions outside of their own. So if you want to use the given program, it's their way or find another program...

If you look at their "proposed" schema, you have redundancy beyond redundancy in regards to the age ratings - they overlap, are inconsistent and just don't make sense (``Everyone and Kids``, as opposed to what makes it different from ``Everyone`` - should it be ``Everyone and dogs ``or maybe ``Everyone and some racial aspect``)
